funnel/assets/js/update.js

```diff
@@ -12,12 +12,13 @@ const Updates = {
     updateTemplate,
     isEditor,
     headerHeight,
+    addReadMore,
   }) {
     Vue.use(VS2);
 
     const updateUI = Vue.component('update', {
       template: updateTemplate,
-      props: ['update', 'iseditor'],
+      props: ['update', 'iseditor', 'addreadmore'],
```

Comment:
This isn't case sensitive?

Reply:
vue template variables(props) don't allow camel case